ivo-projects/todo/src/components/Reminders.js

```diff
@@ -1,20 +1,27 @@
 import React from 'react';
 import moment from 'moment-timezone';
-import EditButtons from './EditButtons';
-import InputField from './InputField';
+import {EditButtons} from './EditButtons';
+import {InputField}  from './InputField';
 
 class RemindersList extends React.Component {
 
     render () {
-        const { reminders, deleteReminder, toggleIsEditingRow } = this.props;
+        const { 
+            reminders, 
+            deleteReminder, 
+            toggleIsEditingRow, 
+            editReminderText 
+        } = this.props;
+
         return (
             <ul className="list-group">
                 {
                   reminders.map((reminder) => (
                         <li key={reminder.id} className="list-group-item">
                           <InputField 
-                            reminderText={reminder.text}
+                            reminder={reminder}
                             isEditing={reminder.isEditing}
```

Comment:
You don't need isEditing and editReminderText -- they should be read from reminder